database.sql

```diff
@@ -655,6 +655,8 @@ CREATE TABLE IF NOT EXISTS `notify` (
 	`seen` tinyint(1) NOT NULL DEFAULT 0,
 	`verb` varchar(255) NOT NULL DEFAULT '',
 	`otype` varchar(16) NOT NULL DEFAULT '',
+	`name_cache` tinytext,
```

Comment:
This is exactly my intention. By adding the documentation in dbstructure.php we could write it into the table definition and could create the documentation from there.